serverless.yml

```diff
@@ -3,13 +3,14 @@ configValidationMode: error
 
 provider:
   name: aws
-  region: ${opt:region, 'us-east-1'}
-  runtime: nodejs10.x
+  region: ${opt:region, 'us-east-1'}  # Default region us-east-1, override with --region
```

Comment:
can we change the 'region' variable to source_region for compatibility with the target_region?

Reply:
Updated